test/helpers.bash

```diff
@@ -123,6 +123,7 @@ for img in "${IMAGES[@]}"; do
 done
 
 function setup_test() {
+    set -u
```

Comment:
Maybe it's better to set it globally (i.e at the beginning of `helpers.bash`)

Reply:
Actually, we don't even need to set this ourselves, if can only we make sure that we really run BATS test using the intended version (v1.3.0 ATM). Ever since commit 04ff588a BATS has set this itself.

I think it would be better to leave this unset here and instead add a BATS test to verify the version of BATS itself (probably forcing it to be the very first test.
WDYT?

Reply:
Actually we should just make sure that the nodes running CI are properly set up (`contrib/test/integration/main.yml` playbook successfully run with `setup` ansible tag) and this whole problem should go away. So I opted for to reverting the added BATS version verification test.